scripts/prepare_env.sh

```diff
@@ -43,8 +43,45 @@ function install_dependencies() {
   curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/"${K8S_VERSION}"/bin/linux/amd64/kubectl && \
   chmod +x kubectl && sudo mv kubectl /usr/local/bin/ && sudo ln /usr/local/bin/kubectl /usr/bin/kubectl || true
 
-  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && \
-  chmod +x minikube && sudo mv minikube /usr/local/bin/ && sudo ln /usr/local/bin/minikube /usr/bin/minikube || true
+  if [[ ! -z "$IS_WSL" && ! -z "$WSL_DISTRO_NAME" ]]; then
```

Comment:
Nits: Consider extract the the common part `! -z "$IS_WSL" && ! -z "$WSL_DISTRO_NAME"` into one boolean variable so we don't need to evaluate it multiple times, and will be more clear.
And please add curly braces around `IS_WSL` and `WSL_DISTRO_NAME` like the other variables do.